lib/matplotlib/pyplot.py

```diff
@@ -286,6 +280,36 @@ class backend_mod(matplotlib.backend_bases._Backend):
                 "framework, as {!r} is currently running".format(
                     newbackend, required_framework, current_framework))
 
+    # Load the new_figure_manager(), draw_if_interactive(), and show()
+    # functions from the backend.
+
+    # Classically, backends can directly export these functions.  This should
+    # keep working for backcompat.
+    new_figure_manager = getattr(backend_mod, "new_figure_manager", None)
+    draw_if_interactive = getattr(backend_mod, "draw_if_interactive", None)
+    show = getattr(backend_mod, "show", None)
```

Comment:
```suggestion
    # draw_if_interactive = getattr(backend_mod, "draw_if_interactive", None)
    # show = getattr(backend_mod, "show", None)
```

These are not used yet so I would prefer to not define them yet.

Reply:
Sure.